samza-test/src/test/java/org/apache/samza/test/framework/AsyncStreamTaskIntegrationTest.java

```diff
@@ -40,16 +43,23 @@ public void testAsyncTaskWithSinglePartition() throws Exception {
     List<Integer> inputList = Arrays.asList(1, 2, 3, 4, 5);
     List<Integer> outputList = Arrays.asList(10, 20, 30, 40, 50);
 
-    CollectionStream<Integer> input = CollectionStream.of("async-test", "ints", inputList);
-    CollectionStream output = CollectionStream.empty("async-test", "ints-out");
+    InMemorySystemDescriptor isd = new InMemorySystemDescriptor("async-test");
+
+    InMemoryInputDescriptor<Integer> imid = isd
```

Comment:
Unrelated to this PR, but should this (and other) class name be TestAsyncStreamTask for consistency with other tests?

Reply:
will do it in a cleanup PR